public/js/app/components/AnalysisMenu.jsx

```diff
@@ -321,34 +313,43 @@ function AnalysisMenu(props) {
       {/* ************************************************************************************ */}
       {/*                               SEMESTER DROPDOWN                          */}
       {/* ************************************************************************************ */}
-      <Dropdown isOpen={dropdownOpen} toggle={toggleDropdown} className="select-semester">
-        <div className="inline-flex padding-top-30">
+
+      <div className="col-4 nopadding">
+      <div className="inline-flex padding-top-30">
           <h3> {translate.select_semester} </h3>
           <InfoButton addClass="padding-top-30" id="info_select_semester" textObj={translate.info_select_semester} />
+      </div>
+      <form>
+        <div className="form-group">
+          <div className="form-select">
+            <div className="select-wrapper">
+              <select 
+              className="form-control"
+              id="semesterDropdownControl" 
+              aria-label={translate.select_semester} 
+              onChange={handleSelectedSemester}>
+                <option>
```

Comment:
- in React important to have `key`, this option is missing key, you can add f.e., key="no-chosen"
- and let's make this option as a placeholder which can have only one value, `value={translate.select_semester}`

Reply:
Very good, I will remember that.